logstreams/src/main/java/io/zeebe/distributedlog/impl/DefaultDistributedLogstreamService.java

```diff
@@ -203,10 +203,6 @@ public long append(long commitPosition, byte[] blockBuffer) {
     }
 
     final ByteBuffer buffer = ByteBuffer.wrap(blockBuffer);
-    return appendBlock(commitPosition, buffer);
-  }
-
-  private long appendBlock(long commitPosition, ByteBuffer buffer) {
     long appendResult = -1;
```

Comment:
Sorry to be nitpicking but I think we could have the declaration inside the try block as well the `lastPosition` assignment and the return. If we block inside the catch then we could have the happy path inside the try instead of split before, inside and after.

Reply:
Everything is fine you're right. I was a bit sloppy yesterday afternoon, since I wanted to get this done :sweat_smile: 
I added a new commit to fix this.